# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../etcnet/0002-iptables-syntax-add-socket-match-extension.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 88ef4d6d752159217c665bb551a09001029f1f88 Mon Sep 17 00:00:00 2001
From: Christian Wiese <christian.wiese@securepoint.de>
Date: Mon, 17 Jan 2011 11:44:03 +0100
Subject: [PATCH 2/2] iptables/syntax: add socket match extension

iptables.8 man page:
---------------------------------------------------------------------------
socket
    This matches if an open socket can be found by doing a socket lookup on
    the packet.
---------------------------------------------------------------------------
---
 etc/net/ifaces/default/fw/iptables/syntax |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/etc/net/ifaces/default/fw/iptables/syntax b/etc/net/ifaces/default/fw/iptables/syntax
index fecef79..26ad286 100644
--- a/etc/net/ifaces/default/fw/iptables/syntax
+++ b/etc/net/ifaces/default/fw/iptables/syntax
@@ -213,6 +213,7 @@ udp-destination-port: -mudp --destination-port
 udp-dest-port: -mudp --destination-port
 udp-dst-port: -mudp --destination-port
 unclean: -munclean
+socket: -msocket
 
 # Parameters
 from: --src
-- 
1.6.6.2

